package.json

```diff
@@ -42,16 +42,20 @@
   "dependencies": {
     "@astrojs/check": "^0.9.4",
     "@astrojs/node": "^9.1.3",
+    "@astrojs/mdx": "^4.2.6",
     "@astrojs/react": "^4.2.1",
     "@nanostores/react": "^0.8.0",
+    "@patternfly/ast-helpers": "^1.4.0-alpha.190",
+    "@patternfly/documentation-framework": "^6.10.4",
```

Comment:
Do we want to bring in the example toolbar as a dependency or copy in the file itself?